internal/app/basin_provider.go

```diff
@@ -18,7 +18,6 @@ type DealInfo struct {
 // BasinProvider ...
 type BasinProvider interface {
 	Create(context.Context, string, string, basincapnp.Schema, common.Address) (bool, error)
-	Push(context.Context, string, string, basincapnp.Tx, []byte) error
```

Comment:
removing `Push` from here as it will not be used now. I left it intact in the capnp definition maybe we can clean that up when we remove capnp otherwise we need to regenerate the files in this PR